### PR TITLE
Support for array of tables. https://github.com/toml-lang/toml/pull/153

### PR DESCRIPTION
**Pull request by mcaruso85**
Hi, 

This support array of tables as specified in that PR. Could you please review and create a new build?

Martin
Thanks !

### CODE REVIEW

**Comment by mcaruso85** [comment]
Hi,

I fixed the lint issues.

Could you please run again?


Thanks !

Martin

**Comment by mcaruso85** [comment]
Hi Kenan,

Is everything ok to publish a new version with this PR? Or there is something missing?

Please let me know, I need this functionality in my project.

Thanks

File: index.js [hunk 1]
```diff
@@ -11,6 +11,26 @@ function format(obj) {
...
 
+function isArrayOfTables(simplePairs) {
+  var isArrayOfTables = false;
+  forEach(simplePairs, function(array) {
+    var value = array[1];
+    if ((Array.isArray(value) && isPlainObject(value[0]))) {
+      isArrayOfTables = true;
+    }
+  });
+  return isArrayOfTables;
```
Comment (KenanY) [inline index.js:22]: This function can be simplified:

```js
function isArrayOfTables(simplePairs) {
  return simplePairs.some(function(array) {
    var value = array[1];
    return Array.isArray(value) && isPlainObject(value[0]);
  });
}
```

**Review by KenanY — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (mcaruso85) [reply index.js:22]: function isArrayOfTables(simplePairs) {
  return simplePairs.some(function(array) {
    return Array.isArray(array[1]) && isPlainObject(array[1][0]);
  });
}

Comment (mcaruso85) [reply index.js:22]: Hi Kenan,

Is that ok?

Martin

Comment (mcaruso85) [reply index.js]: Hi Kenan, 

I updated the code based on your comments. Please review.

Thanks !

**Review by KenanY — APPROVED** [review]

**Comment by KenanY** [comment]
Thank you for this contribution!

Landed as ddd3a702bef4278e43001091169045782919ad4a and published in v2.2.0.

**Pull request closed without merging**